draft-ietf-quic-transport.md

```diff
@@ -2880,6 +2880,28 @@ packet.
 expectations about what implementations do with packets that have errors after
 valid frames? -->
 
+
+## Generating Acknowledgements {#generating-acks}
+
+An acknowledgement SHOULD be sent immediately upon receipt of a second
+ack-eliciting packet. QUIC recovery algorithms do not assume the peer sends
+an ACK immediately when receiving a second ack-eliciting packet.
+
+In order to accelerate loss recovery and reduce timeouts, the receiver SHOULD
+send an immediate ACK after it receives an out-of-order packet. It could send
```

Comment:
What's the exact definition for `out-of-order`? If the last packet number received is `X`, are received packet numbers of `X+2` and `X-1` both considered out of order?

Reply:
Yes, both would be considered out of order.  I thought we already said to send an immediate ACK when the PN is not 1 larger than the largest processed packet number, but I can't find it.